Code/GraphMol/SubstructLibrary/substructLibraryTest.cpp

```diff
@@ -651,6 +651,29 @@ void testPatternHolder() {
 #endif
 }
 
+void testSegFaultInHolder() {
```

Comment:
I think it's worth adding an additional case to this test:
The library contains a mix of reasonable structures and failed structures (along with a check to make sure that the `getMatches()` call returns the correct number of results).